### PR TITLE
FUSETOOLS-2377 - Provide support for 2.18.1-redhat-000015

### PR DESCRIPTION
**Pull request by apupier**
- corresponds to FIS 2.0.0 patch release
- FIS images pointed in launch configuration has been updated too

### CODE REVIEW

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java [hunk 1]
```diff
@@ -56,7 +56,7 @@
 		camelVersionToFuseBOMMapping.put("2.17.3",               "6.3.0.redhat-254");
 	}
 	
-	private static final Set<String> pureFisVersions = Stream.of("2.18.1.redhat-000012").collect(Collectors.toSet());
+	private static final Set<String> pureFisVersions = Stream.of("2.18.1.redhat-000012", "2.18.1.redhat-000015").collect(Collectors.toSet());
```
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java]: we should create a jira to update 10.x branch accordingly

Comment (apupier) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java]: hum we will not rebase the 10.x on the 9.2.x until it is not released?

Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java]: actually I did not want to do another rebase. Thats gonna be too complicated.

Comment (apupier) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java]: means that we have already 2 other commits to cherry-pick :s

**Comment by apupier** [comment]
testPR

**Comment by apupier** [comment]
- All sonar reported issues are part of code that will be removed in 10.0.0
- tests are in success on new CI

**Pull request merged**